framework/src/outputs/Output.C

```diff
@@ -211,6 +220,10 @@ Output::onInterval()
   if (_sync_times.find(_time) != _sync_times.end())
     output = true;
 
+  // check if enough time has passed between outputs
+  if (_time > _last_output_time && _last_output_time + _minimum_time_interval > _time)
```

Comment:
use the timestep tolerance here, with a preferrence for outputting too much

Reply:
Good suggestion